OutOfSchool/OutOfSchool.WebApi.Tests/Services/InstitutionHierarchyServiceTest.cs

```diff
@@ -159,14 +163,20 @@ public async Task GetAll_WhenCalled_ReturnsAllEntities()
             },
         };
 
-        repo.Setup(r => r.GetAll()).ReturnsAsync(expectedEntity);
-        mapper.Setup(m => m.Map<List<InstitutionHierarchyDto>>(expectedEntity)).Returns(expectedDto);
+        repo.Setup(r => r.Get(
+                0,
+                0,
+                string.Empty,
+                It.IsAny<Expression<Func<InstitutionHierarchy, bool>>>(),
+                It.IsAny<Dictionary<Expression<Func<InstitutionHierarchy, dynamic>>, SortDirection>>(),
+                true)).Returns(expectedEntityQueryable);
+
+        mapper.Setup(m => m.Map<List<InstitutionHierarchyDto>>(It.IsAny<InstitutionHierarchy>())).Returns(expectedDto);
 
         // Act
         var result = await service.GetAll().ConfigureAwait(false);
 
         // Assert
-        repo.Verify(r => r.GetAll(), Times.Once);
```

Comment:
Please restore call to `repo.Verify(r => r.Get(...), ..)` or call `repo.VerifyAll()`

Reply:
Done